Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -146,15 +146,39 @@ extension SwiftModule {
         }
     }
 
+    var headerSearchPaths: (key: String, value: String)? {
+        let headerPathKey = "HEADER_SEARCH_PATHS"
+        let headerPaths = dependencies.filter{$0 is CModule}.map{($0 as! CModule).path}
+
+        guard !headerPaths.isEmpty else { return nil }
+
+        if headerPaths.count == 1, let first = headerPaths.first {
+            return (headerPathKey, first)
+        }
+
+        var headerPathValue = "(\n"
+        for header in headerPaths {
+            headerPathValue += "\"\(header)\","
+        }
+        headerPathValue += ")"
```

Comment:
I would suggest just always writing as a list. This can be reduced to:
```
headerPathValue = "( " + headerPaths.map({ "\"\($0)\"" }).joined(separator: ", ") + " )"
```
 Also, we really need utilities or other infrastructure for writing out structured values with appropriate quoting, etc.